test/testrunner.py

```diff
@@ -120,21 +120,19 @@ def pull_dlc_images(images):
         run(f"docker pull {image}", hide="out")
 
 
-def setup_eks_clusters(dlc_images):
+def setup_eks_clusters(framework_name):
```

Comment:
nit: please change to "setup_eks_cluster" as we are enforcing a singular cluster

Reply:
Will do.